doc/zephyr.doxyfile.in

```diff
@@ -810,6 +810,7 @@ EXCLUDE                = @ZEPHYR_BASE@/include/spi_legacy.h \
                          @ZEPHYR_BASE@/include/cmsis_rtos_v2/cmsis_os2.h \
                          @ZEPHYR_BASE@/include/net/coap.h \
                          @ZEPHYR_BASE@/include/net/coap_link_format.h \
+                         @ZEPHYR_BASE@/include/net/sntp.h \
```

Comment:
Why is the SNTP documentation excluded here? (It is added to the list of Doxygen's `EXCLUDE`) We seem to be *replacing* and *not adding* a sockets based SNTP client API here.